src/components/Report/Report.tsx

```diff
@@ -1 +1,139 @@
-export default {}
+import React from 'react'
```

Comment:
We'll need to incorporate the conditional rendering dependent on the alternative year plan choices. Great job!